authors/apps/articles/renderers.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from rest_framework.renderers import JSONRenderer
+
+
+class ArticlesJSONRenderer(JSONRenderer):
+    charset = 'utf-8'
+
+    def render(self, data, media_type=None, renderer_context=None):
+        # If the view throws an error (such as the user can't be authenticated
+        # or something similar), `data` will contain an `errors` key. We want
+        # the default JSONRenderer to handle rendering errors, so we need to
+        # check for this case.
+        errors = data.get('errors', None)
+
+        if errors is not None:
+            pass
+            # As mentioned about, we will let the default JSONRenderer handle
+            # rendering errors.
+            # return super(UserJSONRenderer, self).render(data)
```

Comment:
@abulojoshua1 
- Is this Commented code
- I think you should return the line 
   `return super(ArticlesJSONRenderer, self).render(data)`
I think what that does is return the actual error if the class's `render()`  itself has an error

Reply:
Thanks for Identifying that @ssewilliam, Sharpe eye there! I did that but forgot to get that sorted. I will work on it asap.